.github/workflows/bump_release_version.yml

```diff
@@ -0,0 +1,46 @@
+#############################################################################
+# GitHub Action to bump release version
+#
+#############################################################################
+name: "Bump version"
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  bump:
+    name: Bump version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Latest VERSION
+        id: VERSION_TAG
+        run: |
+          VERSION=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
+          | jq ".tag_name")
+          echo "::set-output name=VERSION::$VERSION"
```

Comment:
this echo is not working , please fix

![image](https://user-images.githubusercontent.com/812984/181258587-64e9155f-e204-40d6-9269-f7347c947dde.png)

Reply:
I am not getting why it is not reading but I can see that it has no usage to read there as we only use this in setting output and we can see the output.

Reply:
Please Google how to do this. We need output